src/operator/isEmpty.ts

```diff
@@ -6,13 +6,13 @@ export function isEmpty(): Observable<boolean> {
   return this.lift(new IsEmptyOperator());
 }
 
-class IsEmptyOperator<T> implements Operator<boolean, boolean> {
-  call (observer: Subscriber<T>): Subscriber<boolean> {
+class IsEmptyOperator implements Operator<any, boolean> {
+  call (observer: Subscriber<boolean>): Subscriber<any> {
     return new IsEmptySubscriber(observer);
   }
 }
 
-class IsEmptySubscriber extends Subscriber<boolean> {
+class IsEmptySubscriber extends Subscriber<any> {
```

Comment:
@blesh `IsEmptySubscriber` isn't actually a subscriber of booleans. It's a `Subscriber` of anything that accepts a `Subscriber` of booleans in its constructor (to which it will pass true or false when it receives its first anything).